low-level/framework/utils/utility.py

```diff
@@ -166,6 +168,34 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
```

Comment:
No new code allowed in generic Utility class anymore. We need to have new functions in specific util classes, like for this we can FileUtils file & class ?